### PR TITLE
disable verbosity for failing chains

### LINKED ISSUE #429: Failing samplers during tests
Tests show sampling failures such as
```
[1] "Error in sampler$call_sampler(args_list[[i]]) : "
[2] "  reached elapsed time limit"                    
[1] "error occurred during calling the sampler; sampling not done"
[1] "Error in sampler$call_sampler(args_list[[i]]) : "
[2] "  reached elapsed time limit"                    
[1] "error occurred during calling the sampler; sampling not done"
[1] "Error in sampler$call_sampler(args_list[[i]]) : "
Execution halted
[2] "  reached elapsed time limit"                    
[1] "error occurred during calling the sampler; sampling not done"
[1] "Error in sampler$call_sampler(args_list[[i]]) : "
[2] "  reached elapsed time limit"                    
[1] "error occurred during calling the sampler; sampling not done"
```

or

```
Warning ('test-estimate_secondary.R:24:1'): (code run outside of `test_that()`)
There were 3402 divergent transitions after warmup. See
https://mc-stan.org/misc/warnings.html#divergent-transitions-after-warmup
to find out why this is a problem and how to eliminate them.
Backtrace:
    ▆
 1. └─EpiNow2::estimate_secondary(...) at test-estimate_secondary.R:24:0
 2.   ├─rstan::sampling(...)
 3.   └─rstan::sampling(...)
 4.     └─rstan (local) .local(object, ...)
 5.       └─rstan:::throw_sampler_warnings(nfit)
```

They occur on all platforms so don't seem to be behind the specific issues on windows (#420) or Ubuntu (#428) but it would probably still be good to avoid theses.

### PR DESCRIPTION
**Pull request by sbfnk**
Together with #432 this should fix #429

### CODE REVIEW

File: R/estimate_infections.R [hunk 1]
```diff
@@ -324,13 +324,12 @@ estimate_infections <- function(reported_cases,
...
   # Initialise fitting by using a previous fit or fitting to cumulative cases
   if (!is.null(args$init_fit)) {
-    if (!inherits(args$init_fit, "stanfit")) {
-      if (args$init_fit %in% "cumulative") {
-        args$init_fit <- init_cumulative_fit(args,
-          warmup = 50, samples = 50,
-          id = id, verbose = FALSE
-        )
-      }
+    if (!inherits(args$init_fit, "stanfit") &&
```
Comment (seabbs) [inline R/estimate_infections.R:327]: just flagging that this and other style changes below should be in their own PR.

Comment (sbfnk) [reply R/estimate_infections.R:327]: I'd normally agree but this was in response to a failing `lintr` check on this PR - without adding it it wouldn't have got the green check mark.

Comment (sbfnk) [reply R/estimate_infections.R:327]: I agree.

**Comment by sbfnk** [comment]
> See note about mixing changes in a single PR.

Thanks for flagging. This was in response to a `lint` failure but agree would have been better first done in its own PR:
ref: https://github.com/epiforecasts/EpiNow2/actions/runs/5626722416?pr=433

**Pull request merged**